### PR TITLE
[Travis] Minor tweak for deploy

### PR DESCRIPTION
**Pull request by ikesyo**
Let's avoid the ambiguity.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -40,4 +40,4 @@ deploy:
   on:
     repo: Carthage/Commandant
     tags: true
-    condition: $JOB = Xcode
+    condition: $JOB = Xcode7.2
```
Comment (mdiep) [inline .travis.yml]: I think we probably want to deploy 7.3 going forward. What do you think?

Comment (ikesyo) [inline .travis.yml]: I'm okay with it :+1:

**Comment by ikesyo** [comment]
Updated :sparkles:

**Comment by mdiep** [comment]
:clap:

**Pull request merged**